### DISCUSSION TITLE
OBS STUDEIO 29.0.2 (64bit) Dont work with OBS-websocket 5.0.1

### DISCUSSION THREAD

**Discussion by Jack220033** [Palakis/obs-websocket#1128] [General]
Hi im trying to insatll OBS-websocket 5.0.1 and i have the error

The following OBS plugings failled to load:

obs-websocket

Please update or remove these plugings.

and of what i read online the 5.0.1 should work with obs 29.0.2 can someone please help me out?
im trying to connect OBS to NOOBS CMDR and i need the websocket
Thanks

**Comment by Jack220033** [comment]
need help

**Comment by tt2468** [comment]
As stated in the "Important Notes" section of the 5.0.1 release, it is not meant to be used for OBS 28.0.0 or above. In 28 we started bundling obs-websocket with OBS by default. To repair your installation, use the `Check For Integrity` tool in the OBS `Help` dropdown.
